traefik/templates/_podtemplate.tpl

```diff
@@ -942,9 +957,24 @@
           {{- toYaml .Values.deployment.additionalVolumes | nindent 8 }}
         {{- end }}
         {{- range $localPluginName, $localPlugin := .Values.experimental.localPlugins }}
+        {{- $pluginType := include "traefik.getLocalPluginType" (dict "plugin" $localPlugin "pluginName" $localPluginName) }}
```

Comment:
Then here we would have something like `$pluginType := default "hostPath" plugin.type`